### DISCUSSION TITLE
How can I try the Issue Forms Beta feature?

### DISCUSSION THREAD

**Discussion by kozo2** [community/community#23412] [Projects and Issues]
<p>I saw that the Issue Forms Beta feature is being utilized here.</p>
<aside class="onebox githubblob" data-onebox-src="https://github.com/pypa/pip/blob/7c3abccd22c4bbd6c1da2ae41845ffbd26c1bb17/.github/ISSUE_TEMPLATE/bug-report.yml">
  <header class="source">

      <a href="https://github.com/pypa/pip/blob/7c3abccd22c4bbd6c1da2ae41845ffbd26c1bb17/.github/ISSUE_TEMPLATE/bug-report.yml" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/pypa/pip/blob/7c3abccd22c4bbd6c1da2ae41845ffbd26c1bb17/.github/ISSUE_TEMPLATE/bug-report.yml" target="_blank" rel="noopener nofollow ugc">pypa/pip/blob/7c3abccd22c4bbd6c1da2ae41845ffbd26c1bb17/.github/ISSUE_TEMPLATE/bug-report.yml</a></h4>

    <pre><code class="lang-yml">name: Bug report
description: Something is not working correctly.
labels: "S: needs triage, type: bug"

body:
  - type: textarea
    attributes:
      label: Description
      description: &gt;-
    validations:
      required: true

  - type: textarea
    attributes:
      label: Expected behavior
      description: &gt;-

  - type: input
</code></pre>

  This file has been truncated. <a href="https://github.com/pypa/pip/blob/7c3abccd22c4bbd6c1da2ae41845ffbd26c1bb17/.github/ISSUE_TEMPLATE/bug-report.yml" target="_blank" rel="noopener nofollow ugc">show original</a>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Is it possible to apply for the use of this beta feature from now on?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>It’s a manual process for now - please send an email to [redacted email]</p>
<p><strong>Edit: Issue forms are now publicly available: <a href="https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/" class="inline-onebox">Issues forms beta for public repositories | GitHub Changelog</a></strong></p>

**Comment by kozo2** [comment]
<p>Thank you for your reply!</p>
<p>I think the email will be sent not from me, but from a community I belong to (called Bioconductor).</p>
<p>Thank you!</p>

**Comment by lukehefson** [comment, marked as answer]
<p>:wave: For any watching this thread, issue forms are now in public Beta for all publicly-accessible repositories!</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">

      <a href="https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/" target="_blank" rel="noopener">The GitHub Blog</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/1469659/181076811-348e86f5-bdc4-4235-917d-ff206e840837.gif" class="thumbnail animated" width="690" height="345"></div>

<h3><a href="https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/" target="_blank" rel="noopener">Issues forms beta for public repositories | GitHub Changelog</a></h3>

  <p>Issues forms beta for public repositories</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>:tada:</p>

**Comment by kozo2** [comment]
<p><a class="mention" href="/lukehefson">@lukehefson</a> Thank you for sharing the information!</p>
